paimon-flink/paimon-flink-common/src/test/java/org/apache/paimon/flink/SchemaChangeITCase.java

```diff
@@ -28,7 +32,211 @@
 /** ITCase for schema changes. */
 public class SchemaChangeITCase extends CatalogITCaseBase {
 
-    // TODO cover more cases once Flink supports more ALTER operations.
```

Comment:
Please change to `TODO cover more cases` because some changes like `AddUniqueConstraint` are still not supported in this PR.